Azure-Samples/communication-services-ios-quickstarts
# Add Chat PushNotification

## Purpose

* Add the sample project "add-chat-push-notification" to guide Contoso user to enable push notification via chat SDK.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Test the code

Step 1: Create a Chat Thread with User A and User B. We will let User A receive push notifications from User B later. 
Step 2: Download this repo and finish all the prerequisites in QuickStart.
Step 3: Put User A’s <ACESS_TOEKN> and your <ACS_RESOURCE_ENDPOINT> into AppSettings.plist. 
Step 4: Set “Enable Bitcode” as “No” for two Pods targets – AzureCommunicationChat and Trouter. 
Step 5: Plug the IOS device into your mac, run the program and click “allow” when asked to authorize push notification on device. You should be able to see “Push Notification Started” printed on Console once the registration is successful. 
Step 6: Let User B sends a message to the thread created in step 1. You (User A) should be able to receive a push notification in your IOS device. 

## What to Check
Verify that the following are valid
N/A

## Other Information
